modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm

```diff
@@ -246,10 +246,10 @@
 		"borg_upgrade_surgicaltools",
 	)
 
-	design_ids -= list(
+/*	design_ids -= list(
 		"borg_upgrade_pinpointer",
 	)
-	return ..()
+	return ..()*/ //BUBBERSTATION REMOVAL
```

Comment:
You can't just remove the return ..() part. You will cause unforseen consequences here.